### ISSUE TITLE
Study systematic contributions to clustering redshift calibration

### ISSUE THREAD

**Issue by tms-epcc** [lsst-uk/photo-redshift-WP3.6#34]
From Oct'22 to Dec' 22 quarterly plan FY22 Annual Evaluation:
Study systematic contributions to clustering redshift calibration

11/JAN/24 - #33 merged with this, that is 
From Oct'22 to Dec' 22 quarterly plan FY22 Annual Evaluation:
Study representativeness of DESI overlap for clustering redshift calibration.

**Comment by tms-epcc** [comment]
17/NOV/22
- cannot be done for quasar but can be for other samples so will progress
- ready to deep-dive

**Comment by tms-epcc** [comment]
13/DEC/22
- examined error bars from software package to see if these are reasonable
- also checked galaxy bias. Identified that this causes some systematic bias for clustering.  
- code is being updated by externals.  Not sure if this will fix these specific errors hence when code ready will need to run and check again

**Comment by tms-epcc** [comment]
@joachimi reported
- this continues to be the major focus.
- Note there is a dependency on a software tool from Bochum that has issues calculating error bars.  This needs 
 to be updated in the DESC implementation of the tool. This is not blocking progress but will need to dealt with in the future.

**Comment by tms-epcc** [comment]
From DEC'22 QU, Jan-Mar'23 plan:
- Continue to assess systematic contributions to clustering redshift calibration with galaxy reference samples, in particular galaxy bias evolution in the photometric sample.

**Comment by tms-epcc** [comment]
02/FEB/23
- @hangqianjun had a meeting with Gianluca who leads the German project
- He is close to finishing yaw (Yet Another WIZZ). Gianluca does not expect much difference for small scale measurements (but quasars will be different)
- at meeting also looked at bias and other results. Gianluca not surprised by results.
- Once software update complete, goal will be to run again with new yaw and DC2 data

**Comment by tms-epcc** [comment]
03/MAR/23
- Gianluca may have finished the new implementation
- so he is currently validating using DC2 dataset from Ellen.
- expecting results form Gianluca in w/b 06/MAR
- will then compare
- code though still being examined in detail

**Comment by tms-epcc** [comment]
11/MAY/23
- Gianluca has some preliminary results with the simulation and updated code
- meeting took place on 10/MAY with the German group to discuss the results and further actions
-  have a general idea of what to include in the project.  Less focussed on systematic and more on demo'ing the clustering redshift performance for Rubin.  This will include using GianLuca's code i.e. YAW code, and run it on DC2 as well as the generated simulation .
 - hence as a first step investigating a more idealised case see new ticket ...

**Comment by tms-epcc** [comment]
29/JUN/23
- the German side had a problem with YAW that has since been solved so progress can now be made

**Comment by tms-epcc** [comment]
20/JUL/23
- unable to meet with Gianluca to progress and discuss the mock results yet

**Comment by tms-epcc** [comment]
10/AUG/23
- unable to discuss mock results with Gianluca 
- he is implementing another RAIL pipeline, once that is finished, expect to him look at the mocks

**Comment by tms-epcc** [comment]
05/OCT/23
- still unable to discuss with Gianluca
- code though is stable but some questions to be answered using LSST mocks 
- salmo mocks to be generated since also useful

**Comment by tms-epcc** [comment]
11/JAN/24
- still not discussed with Gianluca 
- algorithm is operational but implementation at DESC is not yet complete
- expect this will resume when Gianluca returns 
----
-  #33 merged with this, that is,  
From Oct'22 to Dec' 22 quarterly plan FY22 Annual Evaluation:
Study representativeness of DESI overlap for clustering redshift calibration.

**Comment by tms-epcc** [comment]
01/FEB/24 
- Gianluca just back so will organise meeting in near future

**Comment by tms-epcc** [comment]
06/MAR/24
- waiting for Gianluca

**Comment by tms-epcc** [comment]
27/MAR/24
- still waiting for Gianluca
- but priority is to wrap up D3.6.2(#19), after that will push this more.

**Comment by tms-epcc** [comment]
07/MAY/24
- Gianluca starting to implement in RAIL
- from this a working version will be created
- subsequently there will be a meeting with Gianluca

**Comment by tms-epcc** [comment]
29/MAY/24
- brief meeting with Gianluca took place
- caught up on status of YAW and future plans
- Gianluca will continue performance testing of YAW on DC2 mocks
- For Quasars testing, Ellen will produce a mock catalogue for this
- PZ conveners are keen on this

**Comment by tms-epcc** [comment]
26/JUN/24
- YAW has been implemented in RAIL
- will need to send mock catalog to Gianluca for testing (see #60)
- can discuss at RAIL workshop

**Comment by tms-epcc** [comment]
24/JUL/24
- not discussed at workshop but being implemnented in RAIL
- Gianluca contribution finishes end of 2024 so will need to decide what can be done before he leaves and how to take over 
- another group from Mexico, looking at clustering so will discuss with them too

**Comment by tms-epcc** [comment]
18/SEP/24
- visited Bochum and spoke with Gianluca about this
- Gianluca finishes end 2024 and will try to polish the package in advance of this
- so went through YAW package so can take over maintenance
- it is executing locally at UCL
- will have a call with others interested in this package to discuss the handover.  This is proposed to the PZ convener.
- there are others using YAW in DESC but not clear if all are doing the same thing or not hence need the call

**Comment by tms-epcc** [comment]
07/NOV/24 
- depends on #54

**Comment by tms-epcc** [comment]
31/JAN/25
- agreed to close this since covered by #54
